processor/src/test/java/org/mapstruct/ap/test/selection/qualifier/handwritten/Titles.java

```diff
@@ -22,15 +22,16 @@
 
 import org.mapstruct.ap.test.selection.qualifier.annotation.EnglishToGerman;
 import org.mapstruct.ap.test.selection.qualifier.annotation.TitleTranslator;
-//import org.mapstruct.ap.test.selection.qualifier.annotation.TitleTranslator;
 
 import com.google.common.collect.ImmutableMap;
+import org.mapstruct.Named;
 
 /**
  *
  * @author Sjaak Derksen
  */
 @TitleTranslator
+@Named( "TitleTranslator" )
```

Comment:
> Is it the default for all methods declared by this type?

Yes. And.. moreover it is refined by a qualifier on method level. So it's possible to say, for this mapping methods look for `Titles` and translate those from `EghlishToGerman`. Both qualifications need to be met.

Its actually behavior that we specified from the beginning (I did not invent this for the `@Named`.


> Just seeing it for @TitleTranslator, too, btw. I always had it in mind for methods only

:smile: we've got more functionality than you think..